WBooks/Repositories/CommentRepository.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  CommentRepository.swift
+//  WBooks
+//
+//  Created by Megan Cole on 18/03/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import Result
+import UIKit
+import Alamofire
+
+internal class CommentRepository {
+    public func fetchComments(book: Book, onSuccess: @escaping ([Comment]) -> Void, onError: @escaping (Error) -> Void) {
+        let url = URL(string: "https://swift-training-backend.herokuapp.com/books/\(book.id)/comments")!
```

Comment:
- Move the string to a `static let`.
- Add a `guard` clause and remove the `!`.